tests/passes/group-static-promotion/bounded-loop.expect

```diff
@@ -0,0 +1,34 @@
+import "primitives/core.futil";
+component main(@go @static(10) go: 1, @clk clk: 1, @reset reset: 1) -> (@done done: 1) {
+  cells {
+    @external i = std_mem_d1(32, 1, 1);
+    @external j = std_mem_d1(32, 1, 1);
+    lt = std_lt(32);
+    add = std_add(32);
+  }
+  wires {
+    static group incr_i0<1> {
+      i.write_data = add.out;
+      i.addr0 = 1'd0;
+      i.write_en = 1'd1;
+      add.right = i.read_data;
+      add.left = 32'd1;
+    }
+    static group incr_j0<1> {
+      j.write_data = add.out;
+      j.addr0 = 1'd0;
+      j.write_en = 1'd1;
+      add.right = j.read_data;
+      add.left = 32'd1;
+    }
+  }
+
+  control {
+    @bound(5) @static(10) while lt.out {
+      @static(2) seq {
+                @static <1>incr_i0;
```

Comment:
I'm not sure what is going on with the pretty printing here but this looks wrong. Maybe #1418 fixes this?